sdk/core/azure-core/CMakeLists.txt

```
@@ -69,6 +70,7 @@ set(
     inc/azure/core/credentials.hpp
     inc/azure/core/datetime.hpp
     inc/azure/core/exception.hpp
+    inc/azure/core/export.hpp
```

Maybe:
```suggestion
    inc/azure/core/dll_export_windows.hpp
```

Do we expect any customer to directly/individually include this header?

---

Answering your question, we don't expect the customer to ever include this header.
But for the uber-headers per module, the convention is that core.hpp includes ALL headers regardless of how useful they are (this header will get included regardless, because some other headers DO include it)
BTW, regarding file rename, I did as you suggested.